### DISCUSSION TITLE
How do one run MAM4 versus Oslo-Aer eg as NF1850 ?

### DISCUSSION THREAD

**Discussion by MichaelSchulzMETNO** [NorESMhub/noresm3_dev_simulations#6] [Q&A]
It would be nice to keep a discussion here on how to switch between the two aerosol schemes in both 2.3 and 2.5.

**Comment by mvertens** [comment]
**For NorESM2.5 oslo-aero** 
CAM_CONFIG_OPTS must have the flags:
   `camnor -chem trop_mam_oslo`
And this is turned on in a compset via the cam configuration
    `CAM%DEV%LT%NORESM%CAMoslo`

**For NorESM2.5 tropmam4** 
CAM_CONFIG_OPTS must have the flags:
   `-chem ghg_mam4`
 and this is turned on in a compset via the cam configuration
    `CAM%DEV%LT%GHGMAM4`

**NOTE**: that %NORESM turns on noresm physics which in noresm2.5 really does nothing now. And theoretically this oslo-aero and noresm physics are independent. So we should consider introducing a cam compset configuration without %NORESM but with %CAMoslo moving forwards.
